### DISCUSSION TITLE
Instantly reset Menu delay when another Menu is open?

### DISCUSSION THREAD

**Discussion by albertbronsky** [mantinedev/mantine#696] [Q&A] [answered]
In my navbar I am using multiple `<Menu>` components with `trigger="hover"` and `delay` properties.

When user's mouse leaves the Menu the delay timer starts before it disappears.

But when the user's mouse hovers over another `<Menu>`, the old delay timer has not finished yet, thus creating a "stacking" effect with multiple Menus being layered.

Is there a way to instantly finish the delay timer whenever another menu is triggered?

**Comment by rtivital** [comment, marked as answer]
You can fully control Menu opened state – https://mantine.dev/core/menu/#controlled, you can also set `delay` and `transitionDuration` properties to `0` to make component more responsive
